app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -0,0 +1,147 @@
+.stamp-duty-step-one {
+  margin-top: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin-top: $baseline-unit*5;
+  }
+}
+
+.stamp-duty {
+  border: solid #999999 1px;
+  overflow: hidden;
+  padding: 0 $baseline-unit*3.5;
+  max-width: none;
+  margin-top: $baseline-unit;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin-top: $baseline-unit*8;
+  }
+}
+
+.stamp-duty__heading {
+  margin-top: 1.5rem;
+  margin-bottom: 0.5rem;
+  font-size: $baseline-unit*4;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    font-size: $baseline-unit*6.5;
+  }
+}
+
+.stamp-duty__subheading {
+  font-size: 1rem;
+  font-weight: 300;
+  margin-bottom: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    font-size: $baseline-unit*3.5;
+  }
+}
+
+.stamp-duty__form {
+  margin-top: 0;
+  overflow: hidden;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin: 0 0 1rem;
+  }
+}
+
+.stamp-duty__input-description {
+  display: block;
+  font-size: $baseline-unit*2.35;
+  color: black;
+  font-weight: 600;
+  padding: 0.6rem 0.6rem 0.6rem 0;
+  margin: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    float: left;
+    display: inline-block;
+    font-size: 1.3rem;
+    color: black;
+    font-weight: 600;
+
+    padding: 0.6rem 0.6rem 0.6rem 0;
+  }
+}
+
+.stamp-duty__input {
+  border: 1px solid #999999;
+  border-radius: 0.2rem;
+  font-size: 1.3rem;
+  display: inline-block;
+  width: 100%;
+  padding: 0.3rem 0 0.3rem 0.5rem;
+  margin: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    font-size: 1.5rem;
+    width: 45%;
+    float: left;
+    padding: 0.5rem;
+    margin: 0 0.6rem;
+  }
+}
+
+.stamp-duty__input-label {
+  display: inline-block;
+}
+
+.stamp-duty__input input:focus{
+  outline: none;
+}
+
+.stamp-duty__input input {
```

Comment:
This isn't BEM-like either, if we remove the fake-input span then we can just put the class we need on the input in the markup